app/i18n/es/gen.php

```diff
@@ -121,21 +121,21 @@
 		'should_be_activated' => 'JavaScript debe estar activado',
 	),
 	'lang' => array(
-		'cz' => 'Čeština',	// TODO - Translation
-		'de' => 'Deutsch',	// TODO - Translation
-		'en' => 'English',	// TODO - Translation
-		'es' => 'Español',	// TODO - Translation
-		'fr' => 'Français',	// TODO - Translation
-		'he' => 'עברית',	// TODO - Translation
-		'it' => 'Italiano',	// TODO - Translation
-		'kr' => '한국어',	// TODO - Translation
-		'nl' => 'Nederlands',	// TODO - Translation
+		'cz' => 'Checo',
+		'de' => 'Aleman',
+		'en' => 'Ingles',
+		'es' => 'Español',
+		'fr' => 'Frances',
+		'he' => 'Hebreo',
+		'it' => 'Italiano',
+		'kr' => 'Coreano',
+		'nl' => 'Neerlandés',
 		'oc' => 'Occitan',	// TODO - Translation
-		'pt-br' => 'Português (Brasil)',	// TODO - Translation
-		'ru' => 'Русский',	// TODO - Translation
-		'sk' => 'Slovenčina',	// TODO - Translation
-		'tr' => 'Türkçe',	// TODO - Translation
-		'zh-cn' => '简体中文',	// TODO - Translation
+		'pt-br' => 'Portugues (Brasil)',
+		'ru' => 'Ruso',
+		'sk' => 'Eslovaco',
+		'tr' => 'Turco',
+		'zh-cn' => 'Chino Simplificado',
```

Comment:
should we translate those?

Reply:
I'd say probably not; in Dutch I and/or other translators purposefully didn't.